Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/PressableButtonInspector.cs

```diff
@@ -278,10 +281,49 @@ private float DrawPlaneAndHandle(Vector3[] vertices, Vector3 halfExtents, float
             return distance;
         }
 
+        /// <summary>
+        /// Trigger function for plane distance world to/from local space conversion
+        /// </summary>
+        public void onTriggerPlaneDistanceConversion()
```

Comment:
Should this be public method? Is somebody else needing it?

Reply:
we're calling it in the inspector class

Reply:
ah ignore me. no it should be private!